docs/source/reference/yaml-spec.rst

```diff
@@ -818,6 +833,57 @@ OR
       store: gcs
       mode: MOUNT
 
+SkyPilot also supports volumes mount.
+
+You can mount network volumes (e.g. GCP persistent disks, etc.) or instance volumes (e.g. local SSD) to the instances in the cluster.
```

Comment:
I suppose we should require the user to have region specified when launching a cluster when a volume is used?

Reply:
We should also mention that volume is only supported by GCP at the moment? We should also error out if not with GCP?

Reply:
> I suppose we should require the user to have region specified when launching a cluster when a volume is used?
For existing volume, yes, I will add more descriptions.
For newly created volume(no `name` specified), no need to specify region.
> We should also mention that volume is only supported by GCP at the moment? 
This is mentioned in line 828.
> We should also error out if not with GCP?
OK.